app/src/main/java/ca/dal/cs/athletemonitor/athletemonitor/Workout.java

```diff
@@ -0,0 +1,70 @@
+
+package ca.dal.cs.athletemonitor.athletemonitor;
+
+import java.util.ArrayList;
+
+/**
+ * Created by Brent on 2018-02-16.
```

Comment:
Change to 
`Workout class manages the data for a workout, which is a collection of exercises.`